docker/Dockerfile.linux.amd64

```diff
@@ -1,14 +1,24 @@
-FROM alpine:3.6 as alpine
+FROM golang:1.14-alpine as builder
```

Comment:
we prefer our current approach to building the binary outside the dockerfile, and would kindly ask to revert this change.